testing/constraints-3.7.txt

```diff
@@ -6,9 +6,10 @@
 # e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
 # Then this file should have foo==1.14.0
 googleapis-common-protos==1.56.2
-protobuf==3.15.0
+protobuf==3.19.0
```

Comment:
Why not 3.21?

Reply:
I've bumped the version to 3.20.1, the latest 3.x.x version
https://pypi.org/project/protobuf/3.20.1/